apps/website/src/components/shared/form/UploadAttachmentsField.tsx

```diff
@@ -149,6 +152,21 @@ export const useUploadAttachmentsData = (
   return { files: fileReferences, dispatch };
 };
 
+async function handleImageConversion(file: File) {
+  if (
+    file.type === "image/heic" ||
+    file.type === "image/heif" ||
+    file.type === "image/avif"
```

Comment:
I think this condition is redundant given the identical condition wrapping this method call below, but I imagine this can go away completely if my suggestion of mapping image types to a converter makes sense to you